app/src/main/java/com/example/jondhc/yoyo/Level1Activity.java

```diff
@@ -1,23 +1,288 @@
 package com.example.jondhc.yoyo;
 
+import android.content.ClipData;
+import android.media.MediaPlayer;
 import android.os.Bundle;
 import android.support.v7.app.ActionBar;
 import android.support.v7.app.AppCompatActivity;
+import android.view.DragEvent;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.animation.LinearInterpolator;
+import android.view.animation.RotateAnimation;
+import android.widget.TextView;
+
+import java.util.Random;
+import java.util.Timer;
+import java.util.TimerTask;
 
 public class Level1Activity extends AppCompatActivity {
 
+    int basket_counter = 0; //Counter for the fruits
+    public int quantity;    //Quantity of fruits to display
+    public View char_cat;   //Variable for cat imageView
+
+
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        setContentView(R.layout.activity_level1);
+
+        Random randomGenerator = new Random();  //Random generator for fruits number
+        int maxFruits = 9;  //Maximum Number of fruits to display
+        int minFruits = 1;  //Minimum number of fruits to display
```

Comment:
I think one fruit is irrelevant, also this variables could be set as static and outside of the oncreate